### ISSUE TITLE
MultiOgar scrambles my map, any way to fix it?

### ISSUE THREAD

**Issue by CalvinDKL** [CigarProject/Cigar#411]
https://gyazo.com/4c016723656ee61f046ec3f1cdbd110d

So the title pretty much says it, my site has gotten to a point were people are making bots and i am forced to turn up the protection. But then the minimap gets scrambled. its the same as the spectate issue im guessing.

**Comment by CalvinDKHD** [comment]
Yes but maybe a way to let the map slip through?

**Comment by MastaCoder** [comment]
Nope, the whole point of it is to stop all of that. Letting a map through or anything that gives coordinates can be found in just a couple seconds.

So it's pretty much up to you. You can give them a hard time with scramble, or just let them free into it without scramble. (Not saying scramble would help, scramble is now outdated and many bot providers know how to bypass it already.)

**Comment by CalvinDKHD** [comment]
No worries, thanks!
